LICENSE.md

```diff
@@ -1,115 +1,28 @@
-Bedrock Non-Commercial License v1.0
-===================================
+BSD 3-Clause License
 
-Copyright (c) 2011-2021 Digital Bazaar, Inc.
-All rights reserved.
-
-Summary
-=======
-
-This license allows the licensee to use Bedrock and its software modules
-for non-commercial purposes such as self-study, research, personal
-projects, or for evaluation purposes. If the licensee uses Bedrock
-directly or indirectly to generate revenue, or to provide products or
-services to more than 500 people (users), the licensee must immediately
-obtain a non-profit or commercial license.
-
-Examples
-========
-
-These are examples of cases that are allowed by this license:
-
-* The licensee is an individual that creates Bedrock-dependent software for
-  personal use only.
-* The licensee is an individual or group of students/researchers that uses
-  Bedrock to experiment with an idea for a non-commercial project.
-* The licensee is a startup company that prototypes a Bedrock-dependent
-  product before they have cash flow and will be testing the prototype
-  software with less than 500 users. The service will not generate revenue
-  of any kind.
-* The licensee is a for-profit organization that creates a product or
-  service that is used by less than 500 users and is built with or
-  integrates with Bedrock. The service must be exclusively provided for free
-  and no parent, subsidiary, agent, or affiliate organization may profit
-  from its use.
-
-These cases require a non-profit or commercial license:
-
-* The licensee is a non-profit that receives funding to create and/or run a
-  Bedrock-dependent service.
-* The licensee is a startup company with Bedrock-dependent software that is
-  funded by another organization.
-* The licensee is a startup company that is going into production with
-  Bedrock-dependent software.
-* The licensee has more than 500 users using a Bedrock-dependent service
-  either directly or indirectly.
-* The licensee is a medium to large organization that builds or integrates a
-  commercial product or service with Bedrock.
-
-THE LICENSE
-===========
-
-This section and all subsequent sections of this document constitute the
-agreement between the licensee and Digital Bazaar, Inc.
-
-DEFINITIONS
-===========
-
-* Product - The Bedrock software and any modules associated with Bedrock
-where Digital Bazaar, Inc. owns the copyright.
-
-CONDITIONS
-==========
+Copyright (c) 2011-2024, Digital Bazaar, Inc.
```

Comment:
```suggestion
Copyright (c) 2013-2024, Digital Bazaar, Inc.
```

Reply:
Again, there's know way to know which is more correct. My assumption is that @dlongley had put code into Bedrock before he added it to git--based on these dates appearing in files from the beginning.

I tried to leave the dates I found intact to avoid needing to unearth history outside the reach of version control.